acquisitions/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

```diff
@@ -103,21 +102,27 @@ Feature: Tenant object in mod-consortia api tests
     Then status 200
     And match response == { tenants: [{ id: '#(centralTenant)', code: 'ABC', name: 'Central tenants name', isCentral: true }], totalRecords: 1 }
 
+    # verify that 'consortia_configuration' in 'central' tenant has record for 'central' tenant
+    Given path 'consortia-configuration'
+    When method GET
+    Then status 200
+    And match response.centralTenantId == centralTenant
+
   @Negative
   # At this point we have one record in consortium = { id: '#(centralTenant)', code: 'ABC', name: 'Central tenants name', isCentral: true }
   Scenario: Attempt to POST a second tenant with existing 'isCentral' = true, 'id', 'name', 'code'
     # cases for 409
     # attempt to create a second central tenant ('isCentral' = true)
     Given path 'consortia', consortiumId, 'tenants'
-    And param adminUserId = centralAdmin.id
+    And param adminUserId = consortiaAdmin.id
```

Comment:
for central true we don't need adminUserId. we can safely delete it

Reply:
It was mandatory when this tests were added. We have only one admin for tenants in consortia, so just renamed 'centralAdmin' to 'consortiaAdmin' to be more clear.